tensorflow/contrib/layers/python/layers/initializers.py

```diff
@@ -46,8 +46,7 @@ def xavier_initializer(uniform=True, seed=None, dtype=dtypes.float32):
   Args:
     uniform: Whether to use uniform or normal distributed random initialization.
     seed: A Python integer. Used to create random seeds. See
-      [`set_random_seed`](../../../../../api_guides/python/constant_op#random-tensors)
-      for behavior.
+      @{set_random_seed} for behavior.
```

Comment:
Just got the link fixed. I also notice that there is another link to `set_random_seed` so I fixed it too,